src/appleseed/renderer/kernel/shading/closures.cpp

```diff
@@ -2448,6 +2540,8 @@ void register_closures(OSLShadingSystem& shading_system)
 
     register_closure<NPRShadingClosure>(shading_system);
     register_closure<NPRContourClosure>(shading_system);
+
+    register_closure<MatteClosure>(shading_system);
```

Comment:
It looks like other closures are registered in alphabetical order, let's continue the trend (and also move the NPR closures registration to the right places).